### PR TITLE
S1 t2/create sign up page

### PR DESCRIPTION
**Pull request by sehersarin**
**Scrum Board Ticket:** S1 T2: 
**Story:** Create the front-end UI for the Sign-up page
**Description:** Implemented the front-end design component for users who don’t have an account in the system to be able to add their name, email, phone number and password into the system and select their role.
**Testing Criteria:** Tested code by visually inspecting the appearance of each element (ie. Sign-up form, button functionality.) to match designs and observe behaviour as expected.
**How Reviewers can Test Code:**
1. Run locally on http://localhost:3000/.
2. Click on the “Sign-up” button on the Dashboard
3. Observe the Sign-up Form appear.
    1. Users can enter criteria into each of the boxes.
    2. Users can also click on “I already have an account” button to be taken to the Sign-In page.

### CODE REVIEW

File: react-ui/src/components/SignUpForm.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                  name="last_name"
+                  size ="40"
+                  placeholder= "Enter Last Name" 
+                  onChange={this.handleFormChange} />
+                </label>
+                <br></br>
+                <br></br>
+                <label for="role" className ="Title-Style2"> Select Role: </label>
+                <select name="role" className ="InputFields2" id="type">
+                  <option value="student">Student</option>
```
Comment (sehersarin) [inline react-ui/src/components/SignUpForm.js:73]: Included these values in the dropdown for "role" based on the design document. Please let me know if I have missed any worker specialization values.

File: react-ui/src/components/SignUpForm.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+          <Row>
+           <Col sm={12} align="center">
+           <form onSubmit={this.login}>
+           
+                <label>
+                <input 
+                  className ="InputFields" 
+                  type="text" 
+                  name="first_name"
+                  size ="40"
```
Comment (sehersarin) [inline react-ui/src/components/SignUpForm.js:55]: Input sizes limited based off of "create table" statements defined in the design document.

**Comment by prdbondi** [comment]
Was able to test out the interface. Very clean and to specification. Appreciate the hard work that went into this!

**Comment by prdbondi** [comment]
Seher, do you know why there are conflicting files shown? What is the recommended way to close these conflicts?

**Review by prdbondi — CHANGES_REQUESTED** [review]
Seher, let me know if you are able to identify a resolution to the conflicting files in this PR! 
The interface is looking great, I was able to get it up and running.

**Comment by sehersarin** [comment]
Hi Peter, thank you for reviewing. The merge conflicts were showing due to the package.json & package-lock.json files being committed and different versions being installed for various dependencies. I have resolved all the merge conflicts, as per your request, to maintain a stable version of the master branch.

File: server/package.json [hunk 3]
```diff
@@ -10,12 +10,12 @@
   "dependencies": {
     "dotenv": "^4.0.0",
     "express": "^4.16.2",
-    "joi": "^17.2.1",
     "lodash": "^4.17.20",
     "moment": "^2.29.1",
     "morgan": "^1.9.0",
     "pg": "^7.3.0",
     "pg-monitor": "^0.8.3",
+    "pg-native": "^3.0.0",
```
Comment (sehersarin) [inline server/package.json:19]: installed pg-native for mac

**Review by prdbondi — APPROVED** [review]
Thank you for the quick response to my request! Approving

**Pull request merged**